tests/test_mcts.py

```diff
@@ -58,6 +63,45 @@
 
 
 class TestMctsNodes(test_utils.MiniGoUnitTest):
+    def test_puct(self):
```

Comment:
can you give this test a more descriptive name? Not sure what it's doing, other than a general integration-ish test.

Reply:
I updated the name.